README.md

```diff
@@ -1,2 +1,183 @@
-# terraform-azure-backup
-Highly opinionated management of Kubernetes backup with velero on AKS (Azure Kubernetes Service)
+# Terraform management of velero backup on AKS
+
+## Introduction
+
+This module manages velero backup on AKS (Azure Kubernetes Service)
+
+## Usage
+
+Instantiate the module by calling it from Terraform like this:
+
+```hcl
+module "azure-basics" {
+  source  = "dodevops/velero/azure"
+  version = "<version>"
+}
+```
+
+<!-- BEGIN_TF_DOCS -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+The following providers are used by this module:
+
+- azurerm
+
+- helm
+
+- kubernetes
+
+## Modules
+
+No modules.
+
+## Resources
+
+The following resources are used by this module:
+
+- [azurerm_storage_account.storaccbackup](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account) (resource)
+- [azurerm_storage_container.storcontbackup](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_container) (resource)
+- [helm_release.velero](https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release) (resource)
+- [kubernetes_namespace.velero](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/namespace) (resource)
+
+## Required Inputs
+
+The following input variables are required:
+
+### backup\_application\_id
+
+Description: Application ID used for backup
+
+Type: `string`
+
+### backup\_application\_secret
+
+Description: Secret of the backup application
+
+Type: `string`
+
+### backup\_tenant\_id
+
+Description: Tenant ID of the backup application
+
+Type: `string`
+
+### location
+
+Description: Azure location to use for the backup
+
+Type: `string`
+
+### project
+
+Description: Three letter project key
+
+Type: `string`
+
+### resource\_group
+
+Description: Azure Resource Group to use
+
+Type: `string`
+
+### stage
+
+Description: Stage for this ip
+
+Type: `string`
+
+### subscription\_id
+
+Description: The Subscription ID to use
+
+Type: `string`
+
+## Optional Inputs
+
+The following input variables are optional (have default values):
+
+### excluded\_namespaces
+
+Description: A list of namespaces to exclude from velero backup
+
+Type: `list(string)`
+
+Default:
+
+```json
+[
+  "velero",
+  "kube-system",
+  "kube-public",
+  "kube-node-lease"
+]
+```
+
+### included\_namespaces
+
+Description: A list of namespaces to include in velero backup
```

Comment:
What is correct here, title or description? 'included' or 'to include'?

Reply:
What do you mean? "to include" and "included" are basically the same, right?

Reply:
No, not at all.
'included namespaces' implies that namespaces mentioned in this section have been included at the point of the reading.
'to be included' implies that something still has to be included anywhere.
I would not know if I'd have the include the mentioned namespaces at another place to make things work.

Reply:
How about "include_namespaces"? This would fit also to the original velero CLI naming:
`      --include-namespaces stringArray                  Namespaces to include in the backup (use '*' for all namespaces). (default *)`

Reply:
@dploeger Mind that you need to update the README.md. You changed var names in the last two commits, so it needs to be updated. But i guess you want to wait for the outcome of this discussion here?

Reply:
Agreed.